go/enclave/enclave_test.go

```diff
@@ -121,7 +121,7 @@ func gasEstimateSuccess(t *testing.T, w wallet.Wallet, enclave common.Enclave, v
 		t.Fatal(err)
 	}
 
-	if decodeUint64 != 675624 {
+	if decodeUint64 != 676040 {
```

Comment:
this magic number is a bit confusing.
it's better to make it a constant with a comment describing why this particular value is chosen

Reply:
I'm just pulling it out of the debugger whenever I change the management contract's code; @otherview should I be doing something else or is this fine assuming the test is working previously?

Reply:
Nah, that's fine. I think Tudor agrees and means to use a constant instead of a hardcoded value and to add a comment explaining the const.
I'd comment something along the lines of gas estimation is a deterministic process so this value should only change if the data field is changed, changes in the gas estimation algorithm or if something is broken.